js/lib/ui.js

```diff
@@ -194,6 +194,34 @@ elgg.ui.toggleMenu = function(event) {
  * @return void
  */
 elgg.ui.initHoverMenu = function(parent) {
+
+	/**
+	 * For a menu clicked, load the menu into all matching placeholders
+	 *
+	 * @param {String} mac Machine authorization code for the menu clicked
+	 */
+	function loadMenu(mac) {
+		var $all_placeholders = $(".elgg-menu-hover[rel='" + mac + "']");
+
+		// find the <ul> that contains data for this menu
+		var $ul = $all_placeholders.filter('[data-elgg-menu-data]');
+
+		if (!$ul.length) {
+			return;
+		}
+
+		elgg.get('ajax/view/navigation/menu/user_hover/contents', {
+			data: $ul.data('elggMenuData'),
+			success: function(data) {
+				if (data) {
```

Comment:
An error should be shown by elgg.ajax.handleAjaxError, but, yes it'll keep spinning I suppose